src/login/ipban.cpp

```diff
@@ -15,14 +15,14 @@
 #include "login.hpp"
 #include "loginlog.hpp"
 
-// login sql settings
-static char   ipban_db_hostname[64] = "127.0.0.1";
-static uint16 ipban_db_port = 3306;
-static char   ipban_db_username[32] = "ragnarok";
-static char   ipban_db_password[32] = "";
-static char   ipban_db_database[32] = "ragnarok";
-static char   ipban_codepage[32] = "";
-static char   ipban_table[32] = "ipbanlist";
+
+int ipban_db_port = 3306;
+std::string ipban_db_ip = "127.0.0.1";
+std::string ipban_db_id = "ragnarok";
+std::string ipban_db_pw = "";
+std::string ipban_db_db = "ragnarok";
```

Comment:
Please keep the variable names and dont rename them.
And if you could keep the same order it would also be nice.